setup.py

```diff
@@ -330,7 +330,7 @@ def run(self):
 
 if sys.version_info[0] == 3:
```

Comment:
can remove this if since onnx does not support python 2 anymore

Reply:
yep - was removed in the other commit, let me fix the merge conflict